### DISCUSSION TITLE
epoptes client don't work in xubuntu 22.04

### DISCUSSION THREAD

**Discussion by nebro2002** [epoptes/epoptes#208] [Q&A]
I have updated my clients to xubuntu 22.04 with lightdm, but i can't connect with clients, the monitor .
After upgrading i "see" that client (in thumbnail view i see the epoptes default grey icon(std), not client's active screen) I launch epoptes-client -c, i have this :
 epoptes-client -c
Can't use SSL_get_servername
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd
verify error:num=66:EE certificate key too weak
verify return:1
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd
verify error:num=66:EE certificate key too weak
verify return:1
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd
verify return:1
DONE
Successfully fetched certificate from informatico-pro:789

**Comment by alkisg** [comment]
If you login to a client and run `epoptes-client` (without -c), do you then see it in the epoptes server?

**Reply by nebro2002** [reply]
No. I have this :
epoptes-client 
Epoptes-client connecting to informatico-pro:789...

**Comment by alkisg** [comment]
Try once more please. Go to a client, open a terminal, and run:

```
sudo epoptes-client -c
epoptes-client
```

Then upload the whole output here.

**Reply by nebro2002** [reply]
root@siaticsalapadres-pro:~# Epoptes-client connecting to informatico-pro:789...Epoptes-client connecting to informatico-pro:789...^C
root@siaticsalapadres-pro:~# sudo epoptes-client -c
Can't use SSL_get_servername
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd
verify error:num=66:EE certificate key too weak
verify return:1
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd
verify error:num=66:EE certificate key too weak
verify return:1
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd
verify return:1
DONE
Successfully fetched certificate from informatico-pro:789
root@siaticsalapadres-pro:~# epoptes-client 
Epoptes-client connecting to informatico-pro:789...

**Comment by alkisg** [comment]
That's very strange. You should be seeing a "...done" at the end. So... it can get the certificate but it can't communicate while using it?
Are you using some strange firewall? Try to temporarily disable it.
If you want, you may come to Matrix for live support: https://ltsp.org/guides/chat-room/
